modules/ROOT/pages/message-type-source-at-host.adoc

```diff
@@ -0,0 +1,33 @@
+= Source at Host Message Type
+
+The Source at Host message type defines the way in which your backend applications transform outbound transactions and send them to your organization.
+
+Use the following settings to define JSON or XML settings for the Source at Host message type:
+
+[%header%autowidth.spread]
+|===
+|Setting |Description |Default | Required
+|Format a|Application message structure format. Allowed value is `JSON` or `XML`.
+ | |Yes
+|Schema |Imported JSON or XSD schema file for the message type.| |Yes
+|Message type identifier a| Message identifier value used in the dynamic message routing of outbound B2B transactions. Partner Manager automatically generates this identifier from the schema filename. You can modify this value.
+
+Your partner must pass this value when sending messages, files, or both to Partner Manager:
+
+* If you receive the payload via an HTTP-based endpoint, the message type identifier is the relative path in the HTTP request.
+* If you receive the payload via AS2 or via file-based endpoint protocols such as SFTP and FTP, the filename begins with `<message-type-id>_`.
+
+See xref:outbound-message-routing.adoc[Outbound Message Routing] to understand how outbound routing is performed and how your backend applications should pass the message type identifier when sending application messages to Partner Manager endpoints. | |Yes
```

Comment:
Any possible rewording to avoid saying "how your backend applications **should** pass.." 

I'm thinking something like:

"...and how to configure your backend applications to properly pass the message..."